### ISSUE TITLE
Where can we find the code related to the pre-training of MobileNetV2 ?

### ISSUE THREAD

**Issue by fathana** [shinmura0/DCASE2020_Task2_Solution-Anomaly_detection-#3]
For reproducibility issues, does anyone know where can I find the code used to pre-train the MobileNetV2 model with the Softmax Cross Entropy Classification and MixUp Augmentation ?
Thanks in advance!

**Comment by forewill** [comment]
me too

**Comment by q739554534** [comment]
I have the same question
